rdkit/Chem/__init__.py

```diff
@@ -163,6 +163,18 @@ def FindMolChiralCenters(mol, force=True, includeUnassigned=False, includeCIP=Tr
         centers.append((idx, code))
   return centers
 
+import warnings
+def WrapLogs():
```

Comment:
Make sure the old logging functions still work, but add deprecation warnings (unfortunately Python suppresses deprecation warnings by default, so I'm not sure if anyone'll notice).

Reply:
Can you please add something to ReleaseNotes.md for this release for this?
The section header is:
```
## Deprecated code (to be removed in a future release):
```

Reply:
Sure!  Just added the deprecation notes.  Should I also add the new logging control functions to the Highlights section?  I'll call them experimental, in case there are threading bugs we haven't found yet.